python/export_util.py

```diff
@@ -76,16 +77,82 @@ def convert_to_isoformat(
         return property
 
 
-def get_graph(ctx: mgp.ProcCtx) -> List[Union[Node, Relationship]]:
+def to_duration_iso_format(value: timedelta) -> str:
```

Comment:
I think the part with calculating hours/minutes/etc. can be made neater with `divmod` ([example](https://github.com/jayaddison/timedelta-isoformat/blob/main/src/timedelta_isoformat/__init__.py#L163-L179)).